lib/trailblazer/rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module Trailblazer
   module Rails
-    VERSION = "2.1.4"
+    VERSION = "2.1.5"
```

Comment:
I think this change together with Changelog should be on a different commit "Release 2.1.5"

Reply:
We release once this change is merged.